allow paging links in browse reversals and semantic domains browse

hanatgit opened the pull request.

hanatgit committed:
add SQL_CALC_FOUND_ROWS to enable paging for reversal entries

hanatgit committed:
fixed paging in semantic domains

hanatgit committed:
Update browseview_func.php

hanatgit committed:
Update browseview_func.php

hanatgit committed:
Update browseview_func.php

add paging for cloud browse sem doms

hanatgit committed:
Update browseview_func.php

phillip-hopper commented on wordpress/wp-content/plugins/sil-dictionary-webonary/include/browseview_func.php:
```diff
@@ -318,7 +317,7 @@ function displayPagenumbers($chosenLetter, $totalEntries, $entriesPerPage, $lang
 			if($requestname == "semnumber")
 			{
-				$display .= "<li><a href=\"#\" onclick=\"displayEntry('-', '" . $chosenLetter . "', " . ($currentPage - 1) . ");\">" . $prevpage . "</a></li> ";
+				$display .= "<li " . $class . "><a href=\"?semdomain=" . $languagecode . "&semnumber=" . $chosenLetter . "&pagenr=" . ($currentPage - 1) . "\">" . $prevpage . "</a></li> ";
```
The variable `$class` is undefined in this function.

hanatgit replied:
nice catch! removed, since $class is necessary only for current page

phillip-hopper commented on wordpress/wp-content/plugins/sil-dictionary-webonary/include/browseview_func.php:
```diff
@@ -685,21 +684,21 @@ function reversalindex($display, $chosenLetter, $langcode, $reversalnr = "")
-				}
-				$englishWord = $reversal->English;
-				 $display .=  "</div>";
+			if($reversal->English != $englishWord)
```
The variable `$englishWord` is undefined.

hanatgit replied:
I think this was used to keep same head entries from being repeated in successive rows of a tabular display. So I fixed the logic and changed the variable name to $previousEnglishWord for clarity.

hanatgit committed:
Update configuration.php

fixed bug in sql

hanatgit committed:
updates for PR

hanatgit committed:
Update shockingly-simple-favicon.php

fixed deprecated call

hanatgit committed:
Update WP-Render-Blogroll.php

fixed "PHP Deprecated:  has_cap was called with an argument that is <strong>deprecated</strong> since version 2.0.0!"

hanatgit committed:
fix deprecated has_cap

fix "PHP Deprecated:  has_cap was called with an argument that is <strong>deprecated</strong> since version 2.0.0"

phillip-hopper reviewed (approved):
LGTM :+1:
